homeassistant/components/unifiprotect/services.py

```diff
@@ -205,32 +222,61 @@ async def set_chime_paired_doorbells(call: ServiceCall) -> None:
     await chime.save_device(data_before_changed)
 
 
+async def get_user_keyring_info(call: ServiceCall) -> ServiceResponse:
+    """Get the user keyring info."""
+    camera = _async_get_ufp_camera(call)
+    ulp_users = camera.api.bootstrap.ulp_users
+    return {
+        str(index + 1): {
+            "full_name": user.full_name,
+            "user_status": user.status,
+            "key_type": key.registry_type,
+            **({"nfc_id": key.registry_id} if key.registry_type == "nfc" else {}),
+            "user_ulp_id": key.ulp_user,
+        }
```

Comment:
Usually for lists its a nested key like 
```
        return {
            "items": [
                {
                    "summary": item["summary"],
                    "description": item["description"],
                } for item in items
            ],
        }
```

https://developers.home-assistant.io/docs/dev_101_services?_highlight=serviceresponse#response-data

However in this case, I think it would make sense to make the key `str(key.ulp_user)` so its stable and they don't have to enumerate the list to find the value in automations.

Reply:
done